perllib/FixMyStreet/App/Form/Waste/Garden/Sacks/Choice.pm

```diff
@@ -0,0 +1,37 @@
+package FixMyStreet::App::Form::Waste::Garden::Sacks::Choice;
+
+use utf8;
+use HTML::FormHandler::Moose::Role;
+
+has_field container_choice => (
+    type => 'Select',
+    label => 'Would you like to subscribe for bins or sacks?',
+    required => 1,
+    widget => 'RadioGroup',
+);
+
+my %sack_num = (
+    sutton => 20,
+    kingston => 10,
+    merton => 25,
```

Comment:
Is the Merton num used anywhere?

Reply:
It should be used a few lines down - in the display of the sack container line?

Reply:
Oh my mistake - I thought the "Buy a roll of $num sacks..." line was inside the else block.